app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -1,48 +1,83 @@
 import React from 'react'
+import { cloneDeep } from 'lodash'
 
-import SupplementalApplicationContainer from './SupplementalApplicationContainer'
 import appPaths from '~/utils/appPaths'
 import mapProps from '~/utils/mapProps'
-import CardLayout from '../layouts/CardLayout'
 import { mapApplication, mapFieldUpdateComment } from '~/components/mappers/soqlToDomain'
 import { updateApplicationAction } from './actions'
 import { mapList } from '~/components/mappers/utils'
+import CardLayout from '../layouts/CardLayout'
+import SupplementalApplicationContainer from './SupplementalApplicationContainer'
+
+class SupplementalApplicationPage extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      persistedApplication: cloneDeep(props.application) // salefroce current perssited data, frozen copy. latest saved copy.
+    }
+  }
+
 
-const SupplementalApplicationPage = ({ application, statusHistory, formFields, onSubmit, fileBaseUrl }) => {
-  const pageHeader = {
-    title: `${application.name}: ${application.applicant.name}`,
-    breadcrumbs: [
-      { title: 'Lease Ups', link: '/lease-ups' },
-      { title: application.listing.name, link: appPaths.toListingLeaseUps(application.listing.id) },
-      { title: application.name, link: '#' }
-    ]
+  handleSaveApplication = async (application) => {
+    const { persistedApplication } = this.state
+    const synchedApplication = cloneDeep(application)
+
+    synchedApplication.total_monthly_rent = persistedApplication.total_monthly_rent
```

Comment:
I still want a comment here about what is going on in these two lines.

Maybe:
"Monthly rent and preferences are only updated in handleSavePreference below."?